roles/confluent.kafka_connect_replicator/tasks/main.yml

```diff
@@ -0,0 +1,314 @@
+  include_role:
+    name: confluent.kerberos
+  vars:
+    kerberos_group: "{{kafka_connect_replicator_group}}"
+    kerberos_user: "{{kafka_connect_replicator_user}}"
+    kerberos_keytab_path: "{{kafka_connect_replicator_monitoring_interceptor_kerberos_keytab_path}}"
+    kerberos_keytab_destination_path: "{{kafka_connect_replicator_monitoring_interceptor_keytab_path}}"
+    kerberos_handler: "Restart Kafka Connect Replicator"
+  when: kafka_connect_replicator_monitoring_interceptor_kerberos_principal is defined
+
+- name: Create Kafka Connect Replicator Config directory
+  file:
+    path: "{{ kafka_connect_replicator.replication_config_file | dirname }}"
+    state: directory
+    mode: 0750
+    owner: "{{kafka_connect_replicator_user}}"
+    group: "{{kafka_connect_replicator_group}}"
+
+- name: Create Kafka Connect Replicator Log4j Config
+  template:
+    src: kafka-connect-replicator-log4j.properties.j2
+    dest: "{{kafka_connect_replicator.log4j_file}}"
+    mode: 0640
+    owner: "{{kafka_connect_replicator_user}}"
+    group: "{{kafka_connect_replicator_group}}"
+  when: kafka_connect_replicator_custom_log4j|bool
+  notify: Restart Kafka Connect Replicator
+
+- name: Create Kafka Connect Replicator Jolokia Config
+  template:
+    src: kafka-connect-replicator-jolokia.properties.j2
+    dest: "{{kafka_connect_replicator_jolokia_config}}"
+    mode: 0640
+    owner: "{{kafka_connect_replicator_user}}"
+    group: "{{kafka_connect_replicator_group}}"
+  when: kafka_connect_replicator_jolokia_enabled|bool
+  notify: Restart Kafka Connect Replicator
+
+- name: Deploy Kafka Connect Replicator JMX Exporter Config File
+  copy:
+    src: "kafka_connect_replicator.yml"
+    dest: "{{kafka_connect_replicator_jmxexporter_config_path}}"
+    mode: 0640
+    owner: "{{kafka_connect_replicator_user}}"
+    group: "{{kafka_connect_replicator_group}}"
+  when: kafka_connect_replicator_jmxexporter_enabled|bool
+
+- name: Create Kafka Connect Replicator Config
+  template:
+    src: kafka-connect-replicator.properties.j2
+    dest: "{{kafka_connect_replicator.replication_config_file}}"
+    mode: 0640
+    owner: "{{kafka_connect_replicator_user}}"
+    group: "{{kafka_connect_replicator_group}}"
+  notify: Restart Kafka Connect Replicator
+
+- name: Create Kafka Connect Replicator Consumer Config
+  template:
+    src: kafka-connect-replicator-consumer.properties.j2
+    dest: "{{kafka_connect_replicator.consumer_config_file}}"
+    mode: 0640
+    owner: "{{kafka_connect_replicator_user}}"
+    group: "{{kafka_connect_replicator_group}}"
+  notify: Restart Kafka Connect Replicator
+
+- name: Create Kafka Connect Replicator Producer Config
+  template:
+    src: kafka-connect-replicator-producer.properties.j2
+    dest: "{{kafka_connect_replicator.producer_config_file}}"
+    mode: 0640
+    owner: "{{kafka_connect_replicator_user}}"
+    group: "{{kafka_connect_replicator_group}}"
+  notify: Restart Kafka Connect Replicator
+
```

Comment:
there is no task `name: Deploy JMX Exporter Config File`
but there is a jmx exporter config file included in this pr, what is the plan for jmx exporter and replicator?

Reply:
iterative approach would be to release replicator without jmx exporter and add to backlog. if we choose to go this route well need to clean up extraneous files/variables